### PR TITLE
Fix item dupe via /logout & NPC (#2495 issue)

### PR DESCRIPTION
**Pull request by Terrarxxn**
Adds a fix for the item dupe method outlined in #2495

### CODE REVIEW

**Review by hakusaro — APPROVED** [review]

**Comment by hakusaro** [comment]
Seems okay! Thank you for your contribution! Just need another review before we can merge this.

**Comment by QuiCM** [comment]
Can't figure out how to add suggestions from mobile.

Suggesting a minor rephrasing of the error text from
```"Please close NPC window for logging out."```
To
```"Please close NPC windows before logging out"```

File: TShockAPI/Commands.cs [hunk 1]
```diff
@@ -911,6 +911,12 @@ private static void Logout(CommandArgs args)
 				return;
 			}
 
+			if (args.Player.TPlayer.talkNPC != -1)
+			{
+				args.Player.SendErrorMessage("Please close NPC window for logging out.");
```
Comment (hakusaro) [inline TShockAPI/Commands.cs]:
```suggestion
				args.Player.SendErrorMessage("Please close NPC windows before logging out.");
```

**Review by hakusaro — DISMISSED** [review]
I added the suggestion that Chris suggested, since he's on mobile.

**Comment by hakusaro** [comment]
@Terrarxxn are you able to apply the changes that we requested todayish?

**Comment by Terrarxxn** [comment]
> @Terrarxxn are you able to apply the changes that we requested todayish?

yes

**Comment by QuiCM** [comment]
The changelog entry is now under the wrong release version, but happy to fix that up in post if @hakusaro doesn't mind

**Comment by hakusaro** [comment]
@QuiCM sure

**Review by QuiCM — APPROVED** [review]

**Pull request merged**